pkg/genericapiserver/config.go

```diff
@@ -55,6 +55,7 @@ import (
 	utilnet "k8s.io/kubernetes/pkg/util/net"
 	"k8s.io/kubernetes/pkg/util/sets"
 	"k8s.io/kubernetes/pkg/version"
+	"k8s.io/kubernetes/pkg/util/config"
```

Comment:
not super-excited about this as a package dependency... util/config seems super vague